.github/workflows/update-geolite-database.yml

```diff
@@ -0,0 +1,106 @@
+name: Update GeoLite Database
+
+on:
+  workflow_dispatch:
+    inputs:
+      branch:
+        description: 'Target branch against which to create PR'
+        required: false
+        default: 'master'
+
+env:
+  MAXMIND_URL: 'https://download.maxmind.com/app/geoip_download?edition_id=GeoLite2-Country&license_key=${{ env.MAXMIND_LICENSE_KEY }}&suffix=tar.gz'
+  MAXMIND_SHA256_URL: 'https://download.maxmind.com/app/geoip_download?edition_id=GeoLite2-Country&license_key=${{ env.MAXMIND_LICENSE_KEY }}&suffix=tar.gz.sha256'
```

Comment:
The license key will be a secret so we should pull it in as a secret  via `{{ secrets.MAXMIND_LICENSE_KEY }}`